lib/media.js

```diff
@@ -7,6 +7,7 @@
 
 const Cam = require('./cam').Cam
 	, linerase = require('./utils').linerase
+	, request = require('request')
```

Comment:
Not sure we should automatically require request. This is a large library that people may not want included in the bundle. I can only think that it could be passed in as an option when initialising a new camera and then throwing an error below if it isn't included. What do you think @hawkeye64 ?

Reply:
I also had trepidation for this. Unfortunately, I cannot get Axios to work with all my camera (Hikvision and Pelco work, but Axis fails as Axios does not do Digest Realm authentication) and you're right, request is a monster. I think it'd be better as a plug-in, but I have no expreience doing that, so it'd take a while to figure out then refactor it.